builtin/files/plugins/kube2iam/manifests/podmonitor.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.prometheusOperatorPodMonitor }}
```

Comment:
I think that adding a reference to the Prometheus operator pod monitor here could add noise to the code, as it is unrelated to kube2ami.